pages/fiscal-sponsorship/about.js

```diff
@@ -487,7 +487,7 @@ export default function FiscalSponsorship() {
           }
         }}
       />
-      <Footer dark />
+      <Footer dark phoneNumber="+1 (844) 237 2290" phoneNumberHyphen="+1-(844)-237-2290"/>
```

Comment:
Please update the rest too. Thanks!
```suggestion
      <Footer dark phoneNumber="+1 (844) 237 2290" phoneNumberUri="+1-844-237-2290"/>
```